### PR TITLE
Lim edits for Section 3.1

### PR DESCRIPTION
**Pull request by pllim**
What can I say? I read very slow when it is not code...

### CODE REVIEW

File: src/ms.tex [hunk 1]
```diff
@@ -603,45 +603,42 @@ \subsection{Project governance} \label{sec:project-governance}
...
 The APE 0 \citep{ape0} document lays out the principles of this governance
 structure, so we refer the reader to that document for a more thorough
-description, and here we only highlight some key elements. While many of these
-principles were already de facto true or have been discussed organically (and
-have been discussed in earlier papers in this series), the APE 0-based governance
-aims to provide a single place where the community can agree as a starting
-point. With this in mind, it highlights the developer and user community of the
-Astropy Project as the ultimate sources of authority, as well as the core
-principle of ``do-ocracy'': those who do work for the Project (be it coding,
+description, and here we only highlight some key elements. The APE 0-based
```
Comment (pllim) [inline src/ms.tex:623]: I try to shorten this section a little as #85 says this paper is too long but hopefully I didn't change its meaning.

Comment (nmearl) [reply src/ms.tex:623]: This works. 👍

**Review by nmearl — APPROVED** [review]

File: src/ms.tex [hunk 2]
```diff
@@ -603,45 +603,42 @@ \subsection{Project governance} \label{sec:project-governance}
 %Briefly summarize new procedures and governance structure, new CoCo, election
 %process overview, etc.
 
-As part of the process of developing \astropypkg into a long-term sustainable
+As part of the process of developing \astropy into a long-term sustainable
```
Comment (adrn) [inline src/ms.tex:606]: I think this is referring to the package (end of this clause is "product")
```suggestion
As part of the process of developing \astropypkg into a long-term sustainable
```

File: src/ms.tex [hunk 3]
```diff
@@ -603,45 +603,42 @@ \subsection{Project governance} \label{sec:project-governance}
...
 when other mechanisms have failed, and includes powers that either
 require central authority or secrets (e.g., passwords). However, APE 0 also
 charges the Committee to devolve responsibilities and seek community input on
 these items as often as possible.
 
-The first Coordination Committee election under these rules took place in Fall
-2021, electing a mix of prior and new coordination committee members,
-and was contested in the sense of more candidates than available slots. This
+The first CoCo election took place in Fall 2021, electing a mix of prior and new members.
+New roles and voting members were also establish. This
```
Comment (adrn) [inline src/ms.tex:641]:
```suggestion
New roles and voting members were also established. This
```

**Comment by pllim** [comment]
Thanks for cleaning it up!

**Pull request merged**